src/protobuf/v1/observability/observability.proto

```diff
@@ -29,8 +29,9 @@ message Response{
     repeated SysProcFileSummaryData FileData = 7;
     repeated SysNwSummaryData InNwData = 8;
     repeated SysNwSummaryData OutNwData = 9;
-    repeated CiliumSummData IngressData = 10;
-    repeated CiliumSummData EgressData = 11;
+    repeated SysNwSummaryData BindNwData = 10;
```

Comment:
Just a small suggestion. Whenever adding a new entry to protobuf struct, make sure to add it at the end.

Reply:
Changes are made in another PR. Kindly review it. Because this PR has unresolved conflicts.